src/components/VIcon/VIcon.js

```diff
@@ -44,7 +43,10 @@ export default {
       case props.xLarge: fontSize = SIZE_MAP.xLarge
         break
     }
-    data.style = { fontSize, ...data.style }
+
+    if (fontSize !== undefined) {
+      data.style = { fontSize, ...data.style }
```

Comment:
change this to data.style.fontSize = fontSize, no reason to use spread operator here

Reply:
I think the spread operator is being used in case style is not a property of data.